src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap

```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InstructorSearchPageComponent should snap with a student table 1`] = `
+<tm-instructor-search-page
+  fbSessionDataTables={[Function Array]}
```

Comment:
You should populate this too for one of the test cases

Reply:
Interestingly this is not being used in the html template itself

Reply:
I think that part is still under migration? @wkurniawan07